### ISSUE TITLE
Web Interface for HQ Assistant

### ISSUE THREAD

**Issue by ps-one** [mikealmond/hq-trivia-assistant#36]
Hello everyone! I created a web interface for this app (www.hqassistant.com). It runs .php on server side and outputs results into .txt file which is used to display information in browser. Method 2 takes way too long to search for answers. If somebody knows how to speed it up please let me know. Thank you!

**Comment by KnottyCord** [comment]
That's awesome! Thanks mate!

**Comment by manofsteel2017** [comment]
this is so perfect. 
hope you can maintain it since i can't get mine to work.
the bearer code is giving me issue.

**Comment by mauri43** [comment]
works great ! only thing is its pretty slow, sometimes takes longer than the question

**Comment by ps-one** [comment]
Guys, thanks everyone for feedback! Just updated look of the website.

@mauri43 Thanks buddy! I made "Method 2" to implement first as it seemed much more accurate. Unfortunately this method takes long time to output results. I might return the original structure of methods implementation to speed up the process. 

Everyone's feedback and suggestions will be much appreciated!

**Comment by manofsteel2017** [comment]
it looks gorgeous now
hquack failed many times but this is more accurate. 
surprise we don't have lot of bots yet

**Comment by exileskim** [comment]
This website doesnt work? Just shows a timer?

**Comment by mauri43** [comment]
@ps-one UI looks amazing! the timer doesn't work though, during the game it started counting up for the time until next show (the picture I attached is during the live show on 2/9/18)
![screen shot 2018-02-09 at 9 09 09 pm](https://user-images.githubusercontent.com/25308449/36057549-82f946ae-0ddd-11e8-988f-e70b270a5bdd.png)

**Comment by JChazzer** [comment]
Does this display the answers for UK games as well? Even so, it's awesome!

**Comment by ps-one** [comment]
@manofsteel2017 Thanks brother! Your feedback is always appreciated!
@mauri43 Thanks a ton for the attached screenshot. Fixed this bug!
@exileskim Sorry for the bug. Everything should work now.

I'm returning the original structure of functions.php file. Method 1 will be implemented first and should speed up output.

**Comment by bstep1012** [comment]
@ps-one still have issue with countdown never going away

**Comment by jainyankee1993** [comment]
if you are on a different time zone other than EST, the countdown will display hours until 3:00 pm EST or 9:00 pm EST. so if the game is on at 8:00 CST, which it generally is, the countdown will still show one hour left until the game starts.

**Comment by jonahverdon** [comment]
The bot says: "Broadcast ended. Next show on Wednesday, 14-Feb-2018 21:00:00 EST for $2500." Even though there was another game. IDK if it's my fault.

**Comment by tdaddy** [comment]
All it says is "Broadcast Ended! Please come back later!" even while there's a game going on.

**Comment by tombets2424** [comment]
Those messages means it works. Just wait till the host is talking then do php run.php

**Comment by tdaddy** [comment]
@tombets2424 isn't the point of the web interface so we don't have to run it ourselves?

**Comment by tombets2424** [comment]
Yes, my comment before yours was suppose to be on another issue.

**Comment by amberphoenix** [comment]
Can this website be made to run for the UK games too?

**Comment by tombets2424** [comment]
It's the same it just displays est time.

**Comment by amberphoenix** [comment]
i've never seen this website working with the UK game though.... I'll check during tonight's game again, but in general i get the game has ended message.

**Comment by tombets2424** [comment]
Hmmmm... interesting. I wonder why that is? Unless the person that did this is based in the US.

**Comment by tdaddy** [comment]
@amberphoenix the same thing happens to me, and i live in EST time zone

**Comment by tdaddy** [comment]
worked for the 3 pm game today. method 2 is just too slow though

**Comment by tombets2424** [comment]
It will be down to the computer and Internet speed. For me method 1 and 2 loads with 3-4 seconds left.

**Comment by tdaddy** [comment]
@tombets2424 to be fair, i only checked it on the last question because i forgot about, and for that question the answer was displayed after the 10 seconds

**Comment by tdaddy** [comment]
wow. this is horribly inaccurate lol. well it was tonight anyway

**Comment by nolanplatt** [comment]
@ps-one ,
Hi!
I was wondering how exactly you exported the results into .txt file and into web page?

**Comment by AravindhStanley** [comment]
Should we run the script for every question or once for the game?

**Comment by anwillcus** [comment]
the timer is is 1 hour off. It should say 2 hours and 22 minutes.

**Comment by perfectusr** [comment]
it appears to be stuck on "Broadcast Ended! Please come back later!" now

**Comment by nolanplatt** [comment]
Change your time... very simple.

**Comment by nanip9997** [comment]
The timer is not the same according to different time zones and It shows wrong timing better make it clear whether it works or not any Indian player .. feedback is appreciated

**Comment by nanip9997** [comment]
![screenshot_2018-03-04-20-34-44-369_com android chrome](https://user-images.githubusercontent.com/29340053/36947002-0528c166-1fec-11e8-8526-216856bba119.png)
